src/database/models/Launches.ts

```diff
@@ -0,0 +1,74 @@
+import {
+  DataType,
+  Column,
+  Model,
+  Table,
+  CreatedAt,
+  UpdatedAt,
+  ForeignKey,
+  BelongsTo,
+} from "sequelize-typescript"
+
+import { Site } from "@database/models/Site"
+import { User } from "@database/models/User"
+
+@Table({ tableName: "launches", paranoid: true })
+export class Launch extends Model {
+  @Column({
+    autoIncrement: true,
+    primaryKey: true,
+    allowNull: false,
+    type: DataType.BIGINT,
+  })
+  id!: number
+
+  @ForeignKey(() => User)
+  @Column
+  userId!: string
+
+  @BelongsTo(() => User, {
+    onUpdate: "CASCADE",
+    onDelete: "CASCADE",
+  })
+  user!: User
+
+  @ForeignKey(() => Site)
+  @Column
+  siteId!: number
+
+  @BelongsTo(() => Site, {
+    onUpdate: "CASCADE",
+    onDelete: "CASCADE",
+  })
+  site!: Site
```

Comment:
i think doing this only exposes the model on the source (`Launches`), which is fine if our target model (`Site`) wouldn't need to know about it (see [here](https://sequelize.org/docs/v6/core-concepts/assocs/#why-associations-are-defined-in-pairs) for details). 

otherwise, wdyt about adding it on the target model too?

Reply:
8c4b838

Reply:
would it also be useful to associate a user with their associated launches?

Reply:
yup -  d5789ce